src/BaseInput.java

```diff
@@ -1,44 +1,56 @@
 import javax.swing.*;
 import java.io.BufferedReader;
-import java.io.FileNotFoundException;
 import java.io.FileReader;
 import java.io.IOException;
 
 public class BaseInput {
 
-    public static String BaseReader(String id, String pass) {
-        System.out.println(id + pass);
-        boolean flag = false;
-        try{
-            BufferedReader baza = new BufferedReader(new FileReader("/Users/artemlaptev/Desktop/baza.txt"));
+    /**
+     * Reads user information from the database stored in the "baza.txt" file and validates
+     * the input ID and password. If a matching ID is found with the correct password,
+     * displays user information in a dialog box.
+     *
+     * @param inputId The input ID string to be validated.
+     * @param inputPass The input password string to be validated.
+     * @return {@code true} if the input ID and password are valid and match a record in the database,
+     *         {@code false} otherwise.
+     *
+     * @throws RuntimeException if an IOException occurs while reading the database file.
+     */
+    public static boolean readUserInputFromDatabase(String inputId, String inputPass) {
+        System.out.println("Reading from the database for input ID: " + inputId + " and password: " + inputPass);
+        boolean success = false;
+        try {
+            BufferedReader baza = new BufferedReader(new FileReader("baza.txt"));
             String line = baza.readLine();
             String[] list;
-            boolean chek = false;
-            while (line != null){
+            while (line != null) {
                 list = line.split(" ");
-                System.out.println(list[0] + list[1]);
+                String readId = list[0];
+                String readPassword = list[1];
+                System.out.println("Reading the line where ID: " + readId + " and password: " + readPassword);
 
-                if ((list[0].equals(id)) & (list[1].equals(pass))){
```

Comment:
split 2 different cases - user was found, but password doesn't match VS user was found and password matches